### DISCUSSION TITLE
Enable https for dockerised based symfony web application framework.

### DISCUSSION THREAD

**Discussion by kaushalshriyan** [dunglas/symfony-docker#634] [Q&A] [answered]
How do i enable https using Letsencrypt SSL certificate for symfony-docker to be allowed to access from Internet? I have setup symfony-docker by referring to https://github.com/dunglas/symfony-docker. Please guide. Thanks in advance.

**Comment by dunglas** [comment, marked as answer]
It's enabled by default. See https://github.com/dunglas/symfony-docker/blob/main/docs/production.md
